apps/tangle-dapp/components/UnbondingStatsItem/UnbondingStatsItem.tsx

```diff
@@ -0,0 +1,55 @@
+'use client';
+
+import { notificationApi } from '@webb-tools/webb-ui-components';
+import { type FC, Fragment, useMemo } from 'react';
+
+import useUnbondingRemainingErasSubscription from '../../data/NominatorStats/useUnbondingRemainingErasSubscription';
+import { NominatorStatsItem } from '../NominatorStatsItem';
+
+const UnbondingStatsItem: FC<{ address: string }> = ({ address }) => {
+  const {
+    data: unbondingRemainingErasData,
+    error: unbondingRemainingErasError,
+  } = useUnbondingRemainingErasSubscription(address);
+
+  const unbondingRemainingErasTooltip = useMemo(() => {
+    if (unbondingRemainingErasError) {
+      notificationApi({
+        variant: 'error',
+        message: unbondingRemainingErasError.message,
+      });
+    }
+
+    if (!unbondingRemainingErasData?.value1) return null;
+
+    if (unbondingRemainingErasData.value1.length === 0) {
+      return 'You have no unbonding tokens.';
+    }
+
+    const elements = unbondingRemainingErasData.value1.map((era, index) => {
```

Comment:
If the remaining eras could ever be a large amount of eras, then this could be a good candidate for `useMemo`, but if it's always a reasonably small amount then this is perfect

Reply:
Let's keep it like this for now!